tests/test_views.py

```diff
@@ -448,6 +448,35 @@ def test_register_template_page_with_invalid_template_name(self):
         assert_equal(res.status_code, 404)
         assert_in('Template not found', res)
 
+    # Regression test for https://github.com/CenterForOpenScience/osf.io/issues/1478
+    def test_registered_projects_contributions(self):
+        # register a project
+        self.project.register_node(None, Auth(user=self.project.creator), '', None)
+        # get the first registered project of a project
+        url = "/api/v1/project/{0}/get_registrations/".format(self.project._primary_key)
```

Comment:
Use `self.project.api_url_for` to compute URLs.

http://cosdev.readthedocs.org/en/latest/osf/guidelines.html#general